api/.env.template

```diff
@@ -3,5 +3,5 @@ ENV=dev
 CORS_ALLOWED_ORIGINS=http://localhost:3000,http://127.0.0.1:3000
 ALLOWED_HOSTS=localhost,127.0.0.1
 DJANGO_SETTINGS_MODULE=api.settings.dev
-GOOGLE_OAUTH2_KEY=<replace with your own key>
+GOOGLE_OAUTH2_CLIENT_ID=<replace with your own key>
 GOOGLE_OAUTH2_SECRET=<replace with your own secret>
```

Comment:
This can also be changed to `GOOGLE_OAUTH2_CLIENT_SECRET`

Reply:
Good point.